db/migrate/20200317155024_remove_over18_from_screener_answers.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOver18FromScreenerAnswers < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :screener_answers, :over18
```

Comment:
```suggestion
    remove_column :screener_answers, :over18, :string
```

Just a very minor thing. Although we don't plan to get back and recover this attribute, the `change` method will need to now the attribute type for the case of a `down` (rollback). So adding the `string` type is recommended.